packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -344,6 +345,7 @@ const GlobalHeader = ({ className, activeSite }) => {
             <li
               css={css`
                 flex: 1;
+                padding: 0px 16px;
```

Comment:
```suggestion
                padding: 0 1rem;
```

We like to use rems for text. Let's update this to use rems as well (`1rem == 16px`)